modules/ROOT/pages/legacy-vpn-migration.adoc

```diff
@@ -0,0 +1,87 @@
+= Legacy VPN Migration to Anypoint VPN
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+If you are using legacy VPN system, you can now migrate from Legacy VPN to Anypoint VPN on a per-route basis, and delete your legacy VPN from the Anypoint Console UI.
+All VPN connections configured prior to November 2018 are eligible for migration to the new Anypoint VPN solution.
+
+== Prerequisites
+
+=== Check Device Requirements
+
+Before migrating to Anypoint VPN, ensure that your device meets all necessary requirements. See xref:vpn-about.adoc[Anypoint VPN] for more information.
+
+[TIP]
+Actual configuration steps, and functionality may differ based on the type of device you use. If supported on your device, use Border Gateway Protocol (BGP) routing.
+
+=== Verify Anypoint VPN Entitlements
+
+. Log in to Runtime Manager using an account with the required roles and permissions. See xref:vpn-create-arm.adoc#prerequisites[Create an Anypoint VPN Connection] for more information.
+. Navigate to the Business Group where your VPC resides (if not in the parent organization).
+** If the VPNs option is present in the left-hand menu, you have the necessary entitlements to create an Anypoint VPN connection.
```

Comment:
```suggestion
** If the VPNs option is present in the navigation menu, you have the necessary entitlements to create an Anypoint VPN connection.
```

Reply:
I'm not sure this should be a bullet. Maybe just a regular sentence under the step

so 

```
If it's not in the parent organization, navigate to the Business Group where your VPC resides.
+
If the VPNs option is present in the left-hand menu, you have the necessary entitlements to create an Anypoint VPN connection.
```